probe-rs/src/architecture/arm/dp/mod.rs

```diff
@@ -154,6 +154,14 @@ bitfield! {
     ///
     /// After a powerup reset, this bit is `0b0`.
     pub c_dbg_rst_req, set_c_dbg_rst_req: 26;
+    /// Error mode. Indicates the reset behavior of the CTRL/STAT.STICKYERR field.
+    ///
+    /// If true, CTRL/STAT.STICKYERR is cleared when a FAULT response is output.
+    ///
+    /// After a powerup reset, the value of this field is false.
+    ///
+    /// This bit only exists on DPv3. See ADIv6 B2.2.3.
+    pub errmode, set_errmode: 24;
```

Comment:
What happens if we try to write this on non v3? I assume it's fine to do so and it's just DC?

Reply:
According to ADIv5’s documentation for `RES0`, it is `IMPLEMENTATION DEFINED` but regardless of the implementation it must not interfere with the operations.
It also specifies that
```
For any RES0 bit, software:
• Must not rely on the bit reading as 0b0.
• Must use an SBZP policy to write to the bit.
```

Reply:
Maybe we could reflect this in the doc comment? :) So future developers know it's safe to read/write this always but there is no guarantees made if this is not v3? :)

Reply:
Is that phrasing ok?

Reply:
Amazing!